onnx/onnx.proto

```diff
@@ -111,6 +111,11 @@ message ModelProto {
   // This field MUST be present.
   optional int64 ir_version = 1;
 
+  // The version of the operator namespace this model targets.  An operator
```

Comment:
So, I guess this is a MUST-be-present field?

Reply:
Yep, definitely mandatory. But we could grandfather models missing this parameter by setting it to version 1.